includes/class-wpcom-legacy-redirector.php

```diff
@@ -106,14 +116,13 @@ public static function wpcom_legacy_add_redirect_js( $hook ) {
 	}
 
 	/**
-	 * @param string $from_url        URL or path that should be redirected; should have leading slash if path.
+	 * @param string $from_url URL or path that should be redirected; should have leading slash if path.
 	 * @param int|string $redirect_to The post ID or URL to redirect to.
-	 * @param bool $validate          Validate $from_url and $redirect_to values.
+	 * @param bool $validate Validate $from_url and $redirect_to values.
```

Comment:
Start of variable names, and start of descriptions, should align. 

```
	 * @param string     $from_url    URL or path that should be redirected; should have leading slash if path.
	 * @param int|string $redirect_to The post ID or URL to redirect to.
	 * @param bool       $validate    Validate $from_url and $redirect_to values.
```

Reply:
See above